### PR TITLE
Issue #24816 : feat : project template created

### PR DESCRIPTION
**Pull request by pradeepkumarcm96**

### CODE REVIEW

File: src/app/manage-learn/project/project-listing/project-listing.component.ts [hunk 1]
```diff
@@ -460,4 +460,11 @@ export class ProjectListingComponent {
             this.selectedProgram(project);
         })
     }
+
+projectTemplate(){
```
Comment (vishwanath1004) [inline src/app/manage-learn/project/project-listing/project-listing.component.ts:464]: This is not required i think, please check this once

File: src/app/manage-learn/project/project-listing/project-listing.component.html [hunk 2]
```diff
@@ -36,4 +36,9 @@ <h5 *ngIf="selectedFilter" class="sb--card__title projects-labels">{{'FRMELEMNTS
 <ion-button class="view-more" expand="block" (click)="loadMore()"
   *ngIf="projects?.length && count > currentOnlineProjectLength">
   {{ 'FRMELEMNTS_BTN_VIEW_MORE' | translate }}
-</ion-button>
\ No newline at end of file
+</ion-button>
+<ion-footer>
+  <ion-button class="view-more" expand="block"  (click)="projectTemplate()">
+    Project Template
```
Comment (vishwanath1004) [inline src/app/manage-learn/project/project-listing/project-listing.component.html:42]: use translate for labels

File: src/app/manage-learn/project/project-template/project-template.page.ts [hunk 3]
```diff
@@ -0,0 +1,598 @@
...
+      projectId: "",
+    };
+    this.categories=[];
+    this.project.categories.forEach((category: any) => {
+      category.label ? this.categories.push(category.label) : this.categories.push(category.name);
+    });
+    this.sortTasks();
+  }
+
+  sortTasks() {
```
Comment (vishwanath1004) [inline src/app/manage-learn/project/project-template/project-template.page.ts:417]: Make this as common method

File: src/app/manage-learn/project/project-template/project-template.page.ts [hunk 4]
```diff
@@ -0,0 +1,598 @@
...
+        this.project._id,
+      ]);
+    }
+  }
+
+  toggle() {
+    this.showDetails = !this.showDetails;
+  }
+
+  async openPopover(ev: any, task?) {
```
Comment (vishwanath1004) [inline src/app/manage-learn/project/project-template/project-template.page.ts:478]: This is also not required

File: src/app/manage-learn/project/project-template/project-template.page.ts [hunk 5]
```diff
@@ -0,0 +1,598 @@
...
+    popover.onDidDismiss().then((data) => {
+      if (data.data) {
+        this.action(data.data, task);
+      }
+    });
+    return await popover.present();
+  }
+
+
+  action(event, task?) {
```
Comment (vishwanath1004) [inline src/app/manage-learn/project/project-template/project-template.page.ts:508]: this is also dead code

File: src/app/manage-learn/project/project-listing/project-listing.component.html (hunk 2 above)
Comment (pradeepkumarcm96) [reply src/app/manage-learn/project/project-listing/project-listing.component.html:42]: only for testing

**Pull request merged**